docs/content/docs/data-structures/counter.md

```diff
@@ -83,11 +88,11 @@ npm install @fluidframework/shared-counter
 ### Creation
 
 The workflow for creating a `SharedCounter` is effectively the same as many of our other DDSes.
-For an example of how to create one, please see our workflow examples for [SharedMap]({{< relref "map.md#creation" >}}).
+For an example of how to create one, please see our workflow examples for [SharedMap](https://fluidframework.com/docs/data-structures/map/#creation).
 
 ### Incrementing / decrementing the value
 
-Once you have created your `SharedCounter`, you can change its value using the [increment]({{< relref "isharedcounter-interface.md#increment-methodsignature" >}}) method.
+Once you have created your `SharedCounter`, you can change its value using the [increment](https://fluidframework.com/docs/apis/counter/isharedcounter-interface#increment-methodsignature) method.
```

Comment:
I dream of a world where we can write something like `{{< apiref ISharedCounter.increment >}}`. It's fun to dream. :)